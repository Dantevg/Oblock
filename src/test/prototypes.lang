{
	a = {x = 10}
	b = {_Proto = a}
	print b
	print b._Proto
	print b.x
}
