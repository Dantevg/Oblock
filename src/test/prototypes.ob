a = {x = 10}
b = a.clone()
print b
print(b._Protos.1)
print(b.x)
