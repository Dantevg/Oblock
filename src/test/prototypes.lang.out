{_Proto = {x = 10}}
{x = 10}
10
